src/model_repository_manager.cc

```diff
@@ -519,6 +526,50 @@ ModelRepositoryManager::ModelLifeCycle::LiveModelStates(bool strict_readiness)
   return live_model_states;
 }
 
+Status
+ModelRepositoryManager::ModelLifeCycle::StopAllModels()
+{
+  LOG_VERBOSE(1) << "StopAllModels()";
```

Comment:
Is it a debug statement?

Reply:
Yes, I think I should put it to VERBOSE(2)